### ISSUE TITLE
Stream songs by ID

### ISSUE THREAD

**Issue by reyraa** [Muzikie/streamer-archived#4]
We should be able to stream songs using their ID.
Before starting the stream, we should validate the user subscription as explained in #3 . We should also validate the API call as explained in #2 . 
We won't implement the stream transaction validation as explained in #5 , this feature is postponed to v0.2.0.

**Comment by reyraa** [comment]
This issue is already resolved and merged into development
